src/cli/commands/config/replace.js

```diff
@@ -3,7 +3,7 @@ const debug = require('debug')
 const path = require('path')
 const log = debug('cli:config')
 log.error = debug('cli:config:error')
-const utils = require('utils')
+const utils = require('../../utils')
```

Comment:
how was this not failing before? Could you add a test too @JGAntunes ? Thank you!

//cc @xicombd